Dockerfile

```diff
@@ -51,4 +53,9 @@ RUN set -ex; \
         chmod +x /usr/bin/tini; \
     fi
 COPY --from=builder /opt/kroxylicious/kroxylicious-app/target/kroxylicious-app-${KROXYLICIOUS_VERSION}-bin/kroxylicious-app-${KROXYLICIOUS_VERSION}/ /opt/kroxylicious/
+
+RUN groupadd -r -g ${CURRENT_USER_UID} ${CURRENT_USER} && useradd -m -r -u ${CURRENT_USER_UID} -g ${CURRENT_USER} ${CURRENT_USER}
+
+USER ${CURRENT_USER}
```

Comment:
While I understand the motivation here, this containerfile is the basis for the published images so I don't think we want to be using the user creating the container. Maybe the right thing to do is default it to `root` or `185` (slightly mysterious to me where 185 comes from) so that we can override when we want to profile but we maintain the old behaviour at runtime of the project images.

Reply:
I've removed the `USER` directory in the last commit, and use `docker-compose` to set it instead. By default it'll run as ROOT, as 185 is used only in the first image built and not in the second.